### ISSUE TITLE
Bug in auto convert data to date

### ISSUE THREAD

**Issue by vietvudanh** [metabase/metabase#10020]
Metabase version `0.32.5`

When the X axis value is week of format `%Y-W%U`, metabase automatically convert them into date, but it seems the grouping process is too forceful and cause data week 52 of previous year to merge with week 0 if this year. 

Is there a way not to do this?

The data week label:
![Screen Shot 2019-05-22 at 2 31 36 PM](https://user-images.githubusercontent.com/4011756/58155569-538df500-7c9e-11e9-9414-4ca7c130e1c0.png)

This is data when visualzie: The peak is an abnormally high value which is wrong:
![Screen Shot 2019-05-22 at 2 23 36 PM](https://user-images.githubusercontent.com/4011756/58155577-5852a900-7c9e-11e9-96fe-8188757f07e2.png)

**Comment by flamber** [comment]
Hi @vietvudanh 
Which database are you using - and version?
Does it make a difference if you use `%Y-W%V` ?

**Comment by vietvudanh** [comment]
HI @flamber.

For database, I used MySQL, version 5.6. But I highly doubt this is database's problem as the column type is just string, not datetime. Metabase cast the type to datetime when visualize.

I fixed this by using time format:` %Y-%U`, and the problem is gone. My X-axis is now just string of week and value of Y-axis is correct.

Also I have checked the value, it is correct that the peak is just sum of week 52 and week 00 of next year.

![Screen Shot 2019-06-03 at 3 07 17 PM](https://user-images.githubusercontent.com/4011756/58786282-60510800-8611-11e9-8b35-27227806ce68.png)
